chains/evm/calls/utils_test.go

```diff
@@ -53,10 +65,33 @@ func (s *UtilsTestSuite) TestToCallArg() {
 
 func (s *UtilsTestSuite) TestToCallArgWithEmptyMessage() {
 	msg := ethereum.CallMsg{}
-	got := ToCallArg(msg)
+	got := calls.ToCallArg(msg)
 	want := map[string]interface{}{
 		"from": common.HexToAddress(""),
 		"to":   (*common.Address)(nil),
 	}
 	s.Equal(want, got)
 }
+
+func (s *UtilsTestSuite) TestTransactNonceUnlockCall() {
```

Comment:
I think it will be pretty fast and valuable to add the same test but with happy pass flow, to test that nonce get released after success

Reply:
The problem is that `UnlockNonce` is a method on `ClientDispatcher` which has to be mocked, so unfortunately this is the only way to test it (check that `UnlockNonce` gets called)

Reply:
but that is ok, just check that UnlockNonce was called if no error occurred during execution